src/test/model-viewer-base-spec.ts

```diff
@@ -146,6 +146,36 @@ suite('ModelViewerElementBase', () => {
       });
     });
 
+    suite.only('capturing screenshots', () => {
+      let element: ModelViewerElementBase;
+      setup(async () => {
+        element = new ModelViewerElement();
+
+        // Avoid testing our memory ceiling in CI by limiting the size
+        // of the screenshots we produce in these tests:
+        element.style.width = '64px';
+        element.style.height = '64px';
+
+        document.body.appendChild(element);
+
+        const modelLoads = waitForEvent(element, 'load');
+        element.src = assetPath('cube.gltf');
+        await modelLoads;
+      });
+
+      teardown(() => {
+        if (element.parentNode != null) {
+          element.parentNode.removeChild(element);
+        }
+      });
+
+      suite('toDataURL', () => {
+        test('produces a URL-compatible string', () => {
+          expect(/^data\:image\//.test(element.toDataURL())).to.be.true;
```

Comment:
This is a new magical incantation I've never seen; what does it do?

Reply:
JavaScript has regular expression literal syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

The usage here can also be written as: `new RegExp('data\:image\/')`.

I'll assign it to a variable so that it is easier to parse.